src/transforms/babel.js

```diff
@@ -301,3 +307,20 @@ async function getJSXConfig(asset, isSourceModule) {
     };
   }
 }
+
+/**
+ * Generates a babel config for stripping away Flow types.
+ */
+function getFlowConfig(asset, isSourceModule) {
+  if (
+    !isSourceModule &&
+    asset.contents.substring(0, 20).indexOf('@flow') > -1
```

Comment:
Should we ensure that this is inside a comment? like look for `// @flow` or something?

Reply:
The issue is that it can be `// @flow` or `/* @flow */` or any other variation of comment. So either we'd have to use a regex or do a babel walk to just detect this. Which would probably slow things down.

I can change it to a regex or babel walk just not sure about performance

Reply:
Yeah a regex would be ok (on the substring). Should be fast enough.